### ISSUE TITLE
IOS 13 current date not there/visible - urgent help please

### ISSUE THREAD

**Issue by vvusts** [henninghall/react-native-date-picker#113]
**Describe the bug**
NOW I saw it is only when ios13 dark mode is set!

Current date is not visible - or it is white not sure on IOS 13 the same code was working fine on IOS 12.
On image you can see it's only current date I moved year down and even not in focus it's still white/not there.
![unnamed](https://user-images.githubusercontent.com/25580183/65712273-ee586f00-e096-11e9-9237-faf2774bab5e.jpg)

**Expected behavior**
That current date is visible.

**To Reproduce**
```javascript
<DatePicker
                maximumDate={this.props.maximumDate}
                textColor={this.props.pickerColor}
                style={{width: this.props.screenWidth}}
                date={this.state.date}
                mode={this.props.mode}
                locale={this.state.locale}
                onDateChange={this.dateChanged}
              />
```

**Smartphone (please complete the following information):**
 - OS: IOS 13, android work fine
 - React Native version [e.g. 0.59.10]
 - react-native-date-picker version 2.7.0

**Comment by henninghall** [comment]
Hi, thanks for reporting this. You can always remove the textcolor prop to get the default colors, it will solve it before we fix this properly. PR's are always welcome!

**Comment by felixchan** [comment]
+1 I don't have any style or textColor props. Still not visible. @henninghall

**Comment by samslow** [comment]
+ 1 this is only can't see on dark mode on iOS

**Comment by henninghall** [comment]
The issue seems to exist in DatePickerIOS which this picker is based on. Does anyone have the possibility to test this solution? https://github.com/facebook/react-native/issues/26299

**Comment by samslow** [comment]
![image](https://user-images.githubusercontent.com/26738367/66255223-3163c200-e7bc-11e9-808c-9a67732382bb.png)
( above testing image is DarkMode )
i'm testing on new RN project, and result is excellent according to [facebook/react-native#26299](https://github.com/facebook/react-native/issues/26299)

But, i don't know how to add below solution code inner `react-native-date-picker`
```
 if (@available(iOS 13, *)) {
    self.window.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
  }
```

![image](https://user-images.githubusercontent.com/26738367/66255279-d2527d00-e7bc-11e9-8d08-fb8c155101bd.png)

Where can i add that solution code ?

**Comment by abakhtiar** [comment]
> @henninghall i tried to that solution
> 
> ![image](https://user-images.githubusercontent.com/26738367/66255223-3163c200-e7bc-11e9-808c-9a67732382bb.png)
> ( above testing image is DarkMode )
> i'm testing on new RN project, and result is excellent according to [facebook/react-native#26299](https://github.com/facebook/react-native/issues/26299)
> 
> But, i don't know how to add below solution code inner `react-native-date-picker`
> 
> ```
>  if (@available(iOS 13, *)) {
>     self.window.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
>   }
> ```
> 
> ![image](https://user-images.githubusercontent.com/26738367/66255279-d2527d00-e7bc-11e9-8d08-fb8c155101bd.png)
> 
> Where can i add that solution code ?
This is what worked for me:
I added this code in the AppDelegate.m of the **PROJECT**  before return YES; (for me was in didFinishLaunchingWithOptions block!).

```
    if #available(iOS 13.0, *) {
        // Always adopt a light interface style.
        overrideUserInterfaceStyle = .light
    }
```
Then I needed to rebuild the project (cmd+R)
Try to add it in the AppDelegate.m of the project not in the node-module. I think keyboard behaviour is affecting from app wide setting so this configuration will cannot override the main setting.

**Comment by samslow** [comment]
What keyboard behaviour ? 

Do you mean that this is a problem that cannot be solved at the library level?

**Comment by abakhtiar** [comment]
> What keyboard behaviour ?
> 
> Do you mean that this is a problem that cannot be solved at the library level?

by behaviour I mean colour..
As I mentioned, this is my assumption so I'm not sure if it can be done also in lib level.
Please let me know if you could set it up in lib.
Thanks!

**Comment by hiren0412** [comment]
I tried this code in appDelegate.m file, but still Date is not visible in IOS 13 in Dark mode. AnyOne having solution. plzzz post here

**Comment by samslow** [comment]
@hiren0412 please show me your code ? this solution is work for me on iOS13

**Comment by abakhtiar** [comment]
@hiren0412 please try this also:
` if (@available(iOS 13, *)) {
      self.window.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
  }`
just before `return YES;` in appDelegate.m
let me know.

**Comment by Theng** [comment]
For me I don't need anything to do with dark mode so I set this in info.plist

```
<key>UIUserInterfaceStyle</key>
 <string>Light</string>
```

**Comment by ptboyer** [comment]
Suppose that #119 has some relation?

**Comment by fmchan** [comment]
anyone can provide solution for Expo which cannot modify the .h file or plist. thank you

**Comment by mikemalpass** [comment]
I got this to work limiting the UIUserInterfaceStyleLight to just the ViewController with datapicker:

#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v)  ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] != NSOrderedAscending)

if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"13.0")) { self.datePicker.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
}

Credit for MACRO to: https://github.com/carlj/CJAMacros/blob/master/CJAMacros/CJAMacros.h

**Comment by ryanbourneuk** [comment]
I've also got this to work, by changing:

    - (UIView *)view
    {
        return [DatePicker new];
    }

in `RNDatePickerManager.m` to:

    - (UIView *)view
    {
        DatePicker *datePicker = [DatePicker new];
        
        if(@available(iOS 13, *)) {
            datePicker.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
        }

        return datePicker;
    }

For a quick fix (accepting that you'll disable dark mode for the component) that'll work - I'll revisit this when I have more time to see if there's a fix that also allows for dark mode support!

**Comment by ptboyer** [comment]
@henninghall How are we looking with cutting a release for this issue soon? :)

**Comment by henninghall** [comment]
Released a fix for this in `2.7.3`

**Comment by Biplovkumar** [comment]
@henninghall 
![Screenshot 2020-01-20 at 10 55 28 AM](https://user-images.githubusercontent.com/30401136/72700887-56b1eb80-3b73-11ea-9de8-8b47c0650a0d.png)



> Released a fix for this in `2.7.3`

Not available this version.

**Comment by henninghall** [comment]
> @henninghall
> ![Screenshot 2020-01-20 at 10 55 28 AM](https://user-images.githubusercontent.com/30401136/72700887-56b1eb80-3b73-11ea-9de8-8b47c0650a0d.png)
> 
> > Released a fix for this in `2.7.3`
> 
> Not available this version.

This is "react-native-date-picker" library, not "react-native-datepicker" 😅
